godot/Demos/WindGrass/GrassMesh.gd

```diff
@@ -2,16 +2,27 @@ tool
 extends MultiMeshInstance
 
 export var extents := Vector2.ONE
-
+export var spawn_outside_circle:bool := false
```

Comment:
Invalid code results in an error. `:bool := false` is defining the type multiple times (once with :bool, again by implicitly having false as the value with `:=` )